Assets/InputSimulationHelpGuide.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class InputSimulationHelpGuide : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private GameObject ShortcutTip = null;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private bool displayShortcutTipOnStart = true;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private GameObject HelpGuideVisual = null;
+
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        public List<KeyCode> displayShortcut;
+
+        // Start is called before the first frame update
+        void Start()
+        {
+            if (displayShortcutTipOnStart)
+            {
+                ShortcutTip.SetActive(true);
+            }
+            HelpGuideVisual.SetActive(false);
+        }
+
+        // Update is called once per frame
+        void Update()
+        {
+            bool shortcutPressed = true;
```

Comment:
After reading this for some time I finally understood that this will fire as soon as all keys are pressed down, but only once. It would be nice to explain that in a comment :-)